src/components/Login/index.jsx

```diff
@@ -9,35 +9,43 @@ export default class Login extends React.Component {
     this.state = {
       isAuthenticated: false,
       isShowingError: false,
-      userName: "",
-      password: ""
+      errorMessage: "Incorrect credentials",
     };
 
     this.handleLogin = this.handleLogin.bind(this);
   }
 
   updateField(e, field) {
     this.setState({
-      [field]: e.target.value
+      [field]: e.target.value,
     });
   }
 
-  handleLogin() {
-    const { userName, password } = this.state;
+  async handleLogin() {
+    const { email, password } = this.state;
 
-    if (userName === "ziv" && password === "papa") {
-      this.setState(prevState => ({
-        isAuthenticated: !prevState.isAuthenticated
-      }));
+    const { data, status } = await authenticateUser({ email, password });
+
+    if (status === 200) {
+      if (data.id !== 0) {
+        this.setState(prevState => ({
+          isAuthenticated: !prevState.isAuthenticated,
+        }));
+      } else {
+        this.setState(prevState => ({
+          isShowingError: !prevState.isShowingError,
```

Comment:
if it enters this else block, I'm guessing it's because user is authenticated. how about using this instead:
`this.setState({
  isShowingError: false,
  isAuthenticated: true
})`